sdks/desktop/Makefile

```diff
@@ -11,5 +11,5 @@ $(CURDIR)/net_4_x_Mono.Debugger.Soft_test.dll: $(TOP)/mcs/class/lib/net_4_x/test
 
 .PHONY: $(TOP)/mcs/class/lib/net_4_x/tests/net_4_x_Mono.Debugger.Soft_test.dll
 $(TOP)/mcs/class/lib/net_4_x/tests/net_4_x_Mono.Debugger.Soft_test.dll:
-	$(MAKE) TEST_MCS_FLAGS="-d:MONODROID_TEST" PROFILE='net_4_x' -C $(TOP)/mcs/class/Mono.Debugger.Soft test
+	$(MAKE) PROFILE='net_4_x' -C $(TOP)/mcs/class/Mono.Debugger.Soft test
```

Comment:
By removing `-d:MONODROID_TEST`, you are breaking https://github.com/mono/mono/pull/11033/files#diff-b3be8767d253feba4c1f59f296284053R52. If that's intentional you should remove it there as well.

Reply:
This hasn't been addressed.